core/dbt/include/global_project/macros/materializations/test/data.sql

```diff
@@ -0,0 +1,10 @@
+{%- materialization test, default -%}
```

Comment:
I notice that the file name is `materializations/test/data.sql`, and I think this is rightly just `materializations/test.sql`. We'll want to use one & the same materialization for _both_ ~data~ one-off and ~schema~ generic/reusable tests.

Reply:
No, please help yourself! Thanks for taking it for a spin. I'll update the name accordingly :)